lerna-debug.log

```diff
@@ -0,0 +1,1248 @@
+0 silly argv {
+0 silly argv   _: [ 'bootstrap' ],
+0 silly argv   hoist: true,
+0 silly argv   lernaVersion: '3.17.0',
```

Comment:
you should remove this from the repository with `git rm lerna-debug.log`. I would also add this as a line to `.gitignore`, so that we don't accidentally add it in the future.